draft-ietf-httpbis-digest-headers.md

```diff
@@ -531,6 +538,9 @@ Response:
 
 ###  Representation data is not contained in the payload
 
+As there is no content coding applied, the `id-sha` and the `id-sha-256`
```

Comment:
```suggestion
As there is no content coding applied, the `sha-256` and the `id-sha-256`
```

Reply:
1- thanks for the fix :) I mis-typed the algorithm name

2- Here and below some `Content-Encoding: identity` + `sha-256`:

- https://github.com/httpwg/http-extensions/blame/76f77c9ce2e9ffa51eb1fe01c8656b9b41b52584/draft-ietf-httpbis-digest-headers.md#L571-L573
- https://github.com/httpwg/http-extensions/blame/76f77c9ce2e9ffa51eb1fe01c8656b9b41b52584/draft-ietf-httpbis-digest-headers.md#L533-L534

Let me know if that works for you!

Thanks++,
R:

Reply:
Ah yeah I missed those sorry. Changes look good to me, lets see if the issue creator has any further suggestions